code/modules/shuttle/shuttle.dm

```diff
@@ -297,6 +299,15 @@
 	initial_engines = count_engines()
 	current_engines = initial_engines
 
+	for (var/turf/T in all_turfs)
+		for(var/obj/effect/shuttle_sound_location/shuttle_loc in T.contents)
+			sound_source = shuttle_loc
+		if (sound_source)
+			break
+
+	if (sound_source == null)
+		log_runtime("SHUTTLE \"[name]\" DOES NOT HAVE A SOUND SOURCE")
```

Comment:
Why does there have to be a sound source?

Reply:
because it will play anyways but without a source

Reply:
Just have it not play then